src/main/java/frc/robot/config/Config.java

```diff
@@ -141,7 +141,15 @@ private Config() {
             .registerToTable(Config.constantsTable);
     public static FluidConstant<Double> maxYawErrorOuterPortCommand = new FluidConstant<>("Outer Port Command Yaw Error", 3.0)
             .registerToTable(Config.constantsTable);
-    
+
+    // Current limiter values
+    public static final int kPeakCurrentAmps = 60; //Peak current threshold to trigger the current limit
```

Comment:
To follow the convention within the code can you use capitals to define these and remove the `k` prefix?

Reply:
OK, fixed that.